pennylane/ops/qubit.py

```diff
@@ -2293,10 +2295,38 @@ def _matrix(cls, *params):
 
     @staticmethod
     def decomposition(theta, wires):
+        # This decomposition is the "upside down" version of that on p17 of https://arxiv.org/abs/2104.05695
         decomp_ops = [
-            DoubleExcitationPlus(theta / 2, wires=wires),
-            DoubleExcitationMinus(theta / 2, wires=wires),
+            qml.CNOT(wires=[wires[2], wires[3]]),
```

Comment:
Yikes! Makes this building block look less like a fundamental one :sweat_smile: 
Still, not too bad in the end!